### PR TITLE
Feature/initialize

### PR DESCRIPTION
**Pull request by Mehdi-Rh**
# Description 
This is a web application where the user can store his books and keep tack of each one.

## Type of changes

- Make the Books and categories pages.
- Display the list of books and the progress of each.

### CODE REVIEW

**Review by Ugboaja-Uchechi — COMMENTED** [review]
Hi @Mehdi-Rh, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, in which all the linters checks aren't passing. Please make sure you fix all linter errors before resubmitting 🙏 .

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Whoistolu — COMMENTED** [review]
Hi @Mehdi-Rh 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted the wrong PR for the `Bookstore: Initialize project with components` milestone. You submitted the Bookstore styling and it does not meet up with the requirements for the Initialize project with component milestones. E.g: Categories should display only a button with the text "Check status" when creating a book it should render a form with inputs for the title and the author of the book. Kindly take a look at this and submit the right PR for this milestone.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Mehdi-Rh** [comment]
Hi @Whoistolu, thanks for your valuable feedback, because of pressure I didn't read the instruction carefully,

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,22 @@
...
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <Router>
+    <Navbar />
+    <Routes>
+      <Route path="/" element={<BookPage />} />
+      <Route path="/calculator" element={<CategoriesPage />} />
```
Comment (Deepakdanger) [inline src/App.js]: - [x] Kindly put the better route name for `CategoryPage`.

File: src/pages/categories/CategoriesPage.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const CategoriesPage = () => (
+  <div>categories</div>
```
Comment (Deepakdanger) [inline src/pages/categories/CategoriesPage.js]: - [x] **Requirement**
* should display only a button with the text "Check status"

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Mehdi-Rh, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
- [ ] Kindly also include the necessary code to run the project locally in the `getting Started` section in the Readme file.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 3]
```diff
@@ -1,25 +1,22 @@
...
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <Router>
+    <Navbar />
+    <Routes>
+      <Route path="/" element={<BookPage />} />
+      <Route path="/author" element={<AuthorsPage />} />
```
Comment (Ghiftee) [inline src/App.js]: - [x] Like the previous reviewer mentioned, this route should be named ```Categories``` and should lead to the category page according to the requirements.

File: src/components/selectAuthor/SelectAuthor.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  <Dropdown>
+    <Dropdown.Toggle variant="Info" id="lessonPanel">
+      <span id="titleCategory">Author</span>
+    </Dropdown.Toggle>
+
+    <Dropdown.Menu>
+      <Dropdown.Item href="#/action-1">Check status</Dropdown.Item>
+    </Dropdown.Menu>
+  </Dropdown>
+);
```
Comment (Ghiftee) [inline src/components/selectAuthor/SelectAuthor.js]: - [x] There should not be an author page, this should be ```category```  and should display only a button with the text "Check status"

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hi there  @Mehdi-Rh 👋🏽 , 

Great work on making the changes requested by a previous reviewer 👏 
To highlight 👍🏽 
- Your code runs smoothly
- Your app follows the design
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- For this project, you are required to use ```gitflow``` however you are currently using ```GitHub flow```. Kindly refer to this [article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) for a refresher on gitflow and remember to apply it in subsequent projects.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/selectAuthor/SelectAuthor.js (hunk 4 above)
Comment (Mehdi-Rh) [reply src/components/selectAuthor/SelectAuthor.js]: This is not an author page, this is the dropdown "selectCategory" that should be changed into author 
book

**Review by mukhammadrizooff — APPROVED** [review]
Hi @Mehdi-Rh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementation ✔️ 
- Perfect design ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**